resources-build-logic/src/main/kotlin/android-base-convention.gradle.kts

```diff
@@ -8,7 +8,7 @@ configure<BaseExtension> {
     compileSdkVersion(31)
 
     defaultConfig {
-        minSdkVersion(16)
-        targetSdkVersion(30)
+        minSdk = 21
```

Comment:
21? why?

Reply:
Without that line, compose fails with exception, because it can work only with 21 api or higher

Reply:
you should not change this common convention plugin. compose not connected to all projects, but connected to some specific projects. so you should just set minSdk 21 for this specific projects. not for all